### DISCUSSION TITLE
Works on macOS 12.7.6 (Monterey) as well

### DISCUSSION THREAD

**Discussion by fmaida** [d99kris/nchat#278] [General]
I just wanted to confirm that nchat compiles correctly and also works on macOS 12 (Monterey), since the documentation only references a more recent version of the operating system. I haven't had the chance to test compatibility with Telegram, but compatibility with WhatsApp seems perfect to me. Attached is an image of my terminal:

![nchat](https://github.com/user-attachments/assets/2fffb0fd-fe46-48b2-99af-bcd2aaca9b24)

Best regards

**Comment by d99kris** [comment]
Hi @fmaida - thanks for sharing 👍 

Yes, actually I would expect nchat to work on even older versions, although I haven't tested recently. Occassionally I test some different Linux-distro though, so nchat should be fairly portable.

That README section maybe has a slightly misleading header, as many more OS versions will be able to run nchat, but the purpose with that part of the documentation is to share which versions I regurarly test nchat on.
